packages/flakybot/flakybot_test.go

```diff
@@ -215,3 +220,57 @@ func TestJSONMarshalMessage(t *testing.T) {
 		t.Fatalf("JSON Marshal -> Unmarshal got diff:\n%v", diff)
 	}
 }
+
+type fakePublisher struct {
+	called []string
+}
+
+func (p *fakePublisher) publish(_ context.Context, msg *pubsub.Message) (serverID string, err error) {
+	p.called = append(p.called, string(msg.Data))
+	return "", nil
+}
+
```

Comment:
I tend to toss a comment above tests that describes the test case, if I'm not using a framework that has a `describe` block.

Reply:
The test name is used for that in Go. The test tests the thing after `Test` in the function name. If you have more than one test for a given function, you can add more detail then.